example/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
```

Comment:
could we remove this unrequired changes?

Reply:
Not sure I understand what these changes mean.